.deploy-settings.toml

```diff
@@ -0,0 +1,2 @@
+[upload]
+workflowId = 'fast'
```

Comment:
Shouldn't this be `studio`? I mean, we designed that workflow specifically for studio.

Reply:
It was `fast` before and I like to keep it that way on the test systems. 
Why put them under additional high load by selecting an overly complicated, very slow workflow just for the sake of compatibility to the other slow and complicated one which is also not being used :D